devel/juno-1/bootstrap.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+CLEANUP=${CLEANUP:-"0"}
+
+GREEN='\e[0;32m'
+RESET='\e[0m'
+RED='\e[0;31m'
+
+function log {
+    echo -e "${GREEN} /-------------------------------------------------------------------------------------- ${RESET}"
+    echo -e "${GREEN} - $@ ${RESET}\n"
+}
+
+function die {
+    echo -e "${RED} /-------------------------------------------------------------------------------------- ${RESET}"
+    echo -e "${RED} - $@ ${RESET}\n"
+    exit 1
+}
+
+PASSWORD=${PASSWORD:-1234567890}
+CHAIN_ID=${CHAIN_ID:-juno-1}
+MONIKER_NAME="$(hostname)"
+JUNO_HOME="juno_home"
+JUNO_GENESIS_HEIGHT="2578099"
+CHAIN_REPO="https://raw.githubusercontent.com/CosmosContracts/mainnet/main/$CHAIN_ID" && \
+export PEERS="$(curl -s "$CHAIN_REPO/persistent_peers.txt")"
```

Comment:
Could we make all these params configurable for testnet as well?

Reply:
I'll do that.